test/sleep/main.c

```diff
@@ -13,6 +17,23 @@
 #define US          (1000)
 #define NS          (1)
 
+// For child process
+#define OPTION_SLEEP_1S    0
+
+// ============================================================================
+// Global variables
+// ============================================================================
+
+static const int SUCCESS = 1;
+static const int FAIL = -1;
+
+// The time obtained from Occlum is not very precise.
+// Here we take 10 millisecond as the time precision of Occlum.
```

Comment:
Typo: it is not "10 millisecond", it is 25 milliseconds.

Reply:
Fix